src/components/TopBar/TopBar.vue

```diff
@@ -560,6 +560,7 @@ export default {
 		justify-content: center;
 		width: 100%;
 		overflow: hidden;
+		height: $clickable-area;
```

Comment:
Could we instead change the height in the left sidebar? I don't want to risk breaking any layout in the callviews of sidebars, etc just because we change the height somewhere

Reply:
No, this is a regression introduced with the wrapping of the controls in the callview. And here I'm touching only the text, which is not shown anywhere the callview